src/utils.ts

```diff
@@ -0,0 +1,37 @@
+export const objectEntries =
```

Comment:
Please let me know which nodejs version vscode uses so that I can update the tsconfig accordingly and get rid of this whole file because newer version already have `Object.entries`, `Object.fromEntires`, and `Array.prototype.flatMap`.